x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/connector/130_connector_update_index_name.yml

```diff
@@ -125,3 +125,29 @@ setup:
         connector_id: test-connector
 
   - match: { index_name: search-1-test }
+
+
```

Comment:
In case indices can just start by `content-`, should we check that updating an index_name to an index that doesn't start with `content-` should fail?

Reply:
@carlosdelest right, those tests will come as a part of https://github.com/elastic/elasticsearch/pull/117778 PR which actually implements the validation logic

Reply:
Got it - this PR just backports the change to the previous tests 👍 . Thanks!